docker/WildFly/Dockerfile

```diff
@@ -13,6 +13,10 @@ COPY target/dependencies/secrets /opt/frank/secrets
 RUN /home/createCS.sh
 
 FROM wildfly-iaf-test-base
+
+ENV JBOSS_JAVA_SIZING="-Xms64m -Xmx4096m -XX:MetaspaceSize=512M -XX:MaxMetaspaceSize=1024m" \
```

Comment:
Moet de xmx er ook in? Ik zou de Xms op 512 zetten en geen max? Die zou niet hard vast moeten staan maar docker/k8s moeten regelen.

Reply:
Docker kan dat helaas niet, tenzij je Swarm gebruikt, Ik heb het overgenomen van de CI, die het op 4GB had staan.
Ik kan het ook aanpassen naar 512, of aanpasbaar maken met een variabel.

Reply:
Ik denk dat met Java8 dit inderdaad moet via de `-Xmx` anders doet Java standaard 1/4 van het geheugen van de machine of zo. Dat werkt niet zo handing onder Docker. Vanaf Java11 doet de JVM dat beter wanneer onder Docker of K8s wordt gedraaid, voor zover ik weet.

Of een max van 512MB nu genoeg is weet ik niet maar we zullen zien of het omvalt, of niet. 😬

Reply:
Ik heb het lokaal de scenario's gedraaid en het is genoeg.